cwl_runner/README.md

```diff
@@ -0,0 +1,116 @@
+# Run a CWL workflow
+
+## This example demonstrates running a multi-stage workflow on Google Cloud Platform
+
+* The workflow is launched with a bash script that calls the [Google Cloud SDK](https://cloud.google.com/sdk)
+* The workflow is defined using the [Common Workflow Language (CWL)](http://www.commonwl.org)
+* The workflow stages are orchestrated using [cwltool](https://github.com/common-workflow-language/cwltool/tree/master/cwltool)
+
+Here's what the bash script does to run a CWL workflow:
+1. Create a disk
+1. Create a Compute Engine VM with that disk
+1. Run a startup script on the VM
+
+The startup script running on the VM will:
+1. Mount and format the disk
+1. Download input files from Google Cloud Storage
+1. Install Docker
+1. Install cwltool
+1. Run the CWL workflow and wait until completion
+1. Copy output files to Google Cloud Storage
+1. Copy stdout and stderr logs to Google Cloud Storage
+1. Shutdown and delete the VM and disk
+
+__Note that the CWL runner does not use the Pipelines API. If you don't have enough quota, the script will fail; it won't be queued to run when quota is available.__
```

Comment:
__Note that the CWL runner does not use the [Pipelines API](https://cloud.google.com/genomics/reference/rest/v1alpha2/pipelines). If you don't have enough quota, the script will fail; it won't be queued to run when quota is available.__